tests/bash/comp-test-lib.bash

```diff
@@ -180,7 +180,7 @@ _completionTests_complete() {
    COMP_CWORD=$((${#COMP_WORDS[@]}-1))
    # We must check for a space as the last character which will tell us
    # that the previous word is complete and the cursor is on the next word.
-   [ "${cmdLine: -1}" = " " ] && COMP_CWORD=${#COMP_WORDS[@]}
+   [ "${cmdLine: -1}" = " " ] && COMP_CWORD=${#COMP_WORDS[@]} && COMP_WORDS[COMP_CWORD]=''
```

Comment:
Why do we need this?

Reply:
Without it, tests fail, even at head. Perhaps something in Bash has changed out from under us?

If the command line ends in a space, Bash appears to be expecting the last `COMP_WORDS` to be an empty string instead of missing.

The failures look like

> ERROR: "testprog prefix default " should complete to "bear     (an animal) bearpaw  (a dessert) dog unicorn  (mythical)" but we got "default"